config/charts/knative-operator/templates/operator.yaml

```diff
@@ -48,6 +48,10 @@ metadata:
     app.kubernetes.io/component: operator-webhook
     app.kubernetes.io/version: "{{ .Chart.Version }}"
     app.kubernetes.io/name: knative-operator
+  {{- if and .Values.knative_operator.operator_webhook.annotations }}
```

Comment:
I think the annotations under the metadata is not necessarily the same as the annotations on the template/metadata.

Reply:
Just remove this section.